.vscode/extensions.json

```diff
@@ -0,0 +1,24 @@
+{
+  // Please make sure you have the following extensions installed and
```

Comment:
you can ignore the red highlights, this is valid syntax since vscode actually parses the file as json5